Gruntfile.js

```diff
@@ -364,12 +364,11 @@ module.exports = function (grunt) {
                 }]
             }
         },
-        uglify: {
+        terser: {
             options: {
-                banner: "<%= banner %>",
```

Comment:
It looks like terser calls this option `preamble`. see: https://github.com/terser/terser/blob/6b6cb9ff01ab5d94e7fa3bc32981fdcab83474a4/README.md#format-options

Reply:
Aha! I tried to find a reference but couldn't. I'll set that up.

Reply:
Resolved in a9356f0 and 69e9a16.